tests/Unit/Helpers/AttributesTraitTest.php

```diff
@@ -33,7 +33,7 @@
 	$attributes['buttonText'] = 'left';
 
 	Components::checkAttr('buttonAlign', $attributes, $manifest);
-})->throws(Exception::class, 'buttonAlign key does not exist in the button block manifest. Please check your implementation.');
+})->throws(Exception::class, 'buttonAlign key does not exist in the button block manifest. If you are using additional components, check if you used the correct block/component prefix in your attribute name. Please check your implementation.');
```

Comment:
better sentence order?
```suggestion
})->throws(Exception::class, 'buttonAlign key does not exist in the button block manifest. Please check your implementation. If you are using additional components, check if you used the correct block/component prefix in your attribute name.');
```

Reply:
You'd have to also change the exception message in the exception itself otherwise the test would fail.

Reply:
tnx, changed in both places